expense-notifications-functions/lib/index.js

```
@@ -0,0 +1,404 @@
+        }
+        logger.info(`Expense updated notification sent successfully. Expense Data: ${JSON.stringify(newExpenseData)}`);
+    }
+    catch (error) {
+        logger.error('Error in onExpenseUpdated function:', error);
+    }
+});
+// Cloud Function to handle expense deletion and notify users
+exports.onExpenseDeleted = (0, firestore_2.onDocumentDeleted)({ document: 'groups/{groupId}/expenses/{expenseId}' }, async (event) => {
+    var _a;
+    try {
+        const expenseData = (_a = event.data) === null || _a === void 0 ? void 0 : _a.data();
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)